### ISSUE TITLE
Supporting Kafka versions and listing ACL

### ISSUE THREAD

**Issue by BestChinchilla** [provectus/kafka-ui#1767]
What versions of Kafka are supported?
Is listing topic ACLs supported?

I installed kafka-ui via docker-compose on on-prem Kafka 3.1.0 with SASL_SSL + PLAIN and user is in super.users list, but I can't list ACLs of my topics

OS: Oracle Linux Server release 8.5
Docker: 20.10.13
docker-compose: v2.3.3

compose.yml:
```
---
version: "3.4"
services:
  kafka-ui:
    container_name: kafka-ui
    image: provectuslabs/kafka-ui:latest
    ports:
      - 8080:8080
    environment:
      KAFKA_CLUSTERS_0_NAME: kafka-cluster
      KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL: SASL_SSL
      KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: hostname.domain.local:9093 # SSL LISTENER!
      KAFKA_CLUSTERS_0_ZOOKEEPER: hostname.domain.local:2181
      KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_LOCATION: /client.truststore.jks
      KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_PASSWORD: password
      KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM: PLAIN
      KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG: 'org.apache.kafka.common.security.plain.PlainLoginModule required username="admin" password="admin-secret";'
    volumes:
      - /opt/kafka/ssl/client.truststore.jks:/client.truststore.jks
```

**Comment by iliax** [comment]
Hi @BestChinchilla. 
What do you mean under "can't list ACLs of my topics" ? Currently kafka-ui doesn't provide ACLs viewing/editing. We plan to implement it in next release.

**Comment by Haarolean** [comment]
Hey @BestChinchilla , thanks for reaching out.

>What versions of Kafka are supported?
Can't give you a determined list, but your case (3.1.0) is supported.

>Is listing topic ACLs supported?
As @iliax said, that's gonna be implemented in 0.5 (#124). Feel free to leave a reaction in that issue so we can prioritise it.
If you have any questions, don't hesitate to ask them either here or in our discord (the link is available in readme).
